android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/remote/SyncL2ROnlyPerformanceTestDefinitions.kt

```diff
@@ -0,0 +1,298 @@
+package com.mongodb.stitch.android.services.mongodb.remote
+
+import android.util.Log
+import com.google.android.gms.tasks.Tasks
+import com.mongodb.stitch.core.services.mongodb.remote.sync.DefaultSyncConflictResolvers
+import org.bson.Document
+
+import com.mongodb.stitch.core.services.mongodb.remote.ExceptionListener
+import org.bson.types.ObjectId
+
+class SyncL2ROnlyPerformanceTestDefinitions {
+    companion object {
+        private val TAG = SyncL2ROnlyPerformanceTestDefinitions::class.java.simpleName
+
+        // NOTE: Many of the tests above 1024 bytes and above 500 docs will fail for various
+        // reasons because they hit undocumented limits. These failures along with stacktraces will
+        // be present in the reported results
+        private val docSizes = intArrayOf(1024, 2048, 5120, 10240, 25600, 51200, 102400)
+        private val numDocs = intArrayOf(100, 500, 1000, 5000, 10000, 25000)
+
+        fun testInitialSync(testHarness: SyncPerformanceIntTestsHarness, runId: ObjectId) {
+            val testName = "testL2R_InitialSync"
+            Log.d(TAG, testName)
+
+            val params = TestParams(
+                    runId = runId,
+                    testName = testName,
+                    dataProbeGranularityMs = 400L,
+                    docSizes = docSizes,
+                    numDocs = numDocs,
+                    numIters = 3,
+                    numOutliersEachSide = 0,
+                    outputToStitch = true,
+                    stitchHostName = "https://stitch.mongodb.com"
```

Comment:
[q] shouldn't this be configurable?

Reply:
sure. I'll make this another local property like the Stitch API key. honestly all of these should be fed into local.properties, but we can iterate as we add more test definitions.